### PR TITLE
cleanup airbyte_configs table

### PR DESCRIPTION
**Pull request by subodh1810**
Issue : https://github.com/airbytehq/airbyte-internal-issues/issues/342

---
This change is [Reviewable](https://reviewable.io/reviews/airbytehq/airbyte/9437)

### CODE REVIEW

**Review by cgardens — APPROVED** [review]
great!

File: airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java [hunk 1]
```diff
@@ -4,22 +4,23 @@
...
+import java.util.Set;
 import java.util.function.Function;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ConfigsDatabaseInstance extends BaseDatabaseInstance implements DatabaseInstance {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ConfigsDatabaseInstance.class);
-
+  private static final Set<String> INITIAL_EXPECTED_TABLES = Collections.singleton("airbyte_configs");
```
Comment (davinchia) [inline airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java:23]: why do we still need this?

**Review by davinchia — COMMENTED** [review]
Nice!

One small comment.

Other than that, should we also update the migration version in the docker and kube env files?

File: airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java (hunk 1 above)
Comment (subodh1810) [reply airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java:23]: @davinchia So the way Flyway is setup is that first we launch a fresh instance of Airbyte,  we first create `airbyte_configs` table and then run all the migrations and eventually reach the latest state. So thats why this is required, we expect this table to be present to determine if the DB has been initialised

**Comment by subodh1810** [comment]
@davinchia regarding this, we can not update the migration version in this PR. It should be done as part of Airbyte release cause if we do it in this PR, if someone does a git pull in airbtye repo and tries to launch Airbyte, it wont work 

> Other than that, should we also update the migration version in the docker and kube env files?

File: airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java (hunk 1 above)
Comment (davinchia) [reply airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java:23]: aye, I get it. annoying. can you leave a comment then? it's not obvious from someone reading code. thanks.

**Comment by davinchia** [comment]
@subodh1810 got it! are you going to release after we merge this in?

**Review by davinchia — APPROVED** [review]

**Pull request merged**